index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <title>
+    Ariel's Portfolio
+    </title>
+    <section id="one">
```

Comment:
section id name needs to be more descriptive eg: homeContent

Reply:
Done